### ISSUE TITLE
Some Errors with RedstoneConduitNetwork

### ISSUE THREAD

**Issue by MrRiegel** [SleepyTrousers/EnderIO#3429]
Hi,
I got some errors with my mod and every time RedstoneConduitNetwork is in the stack trace and the stack trace is different each time.
And the error is java.lang.Error, so very hard to investigate.

What am I doing wrong?

https://github.com/MrRiegel/Storage-Network/issues/55

**Comment by HenryLoenwind** [comment]
I have to guess here: You are doing things in you block update which can trigger other block updates. It usually is a good idea to just record the update and act on your next tick, like the redstone conduit does.

But as I said, pure guessing.

**Comment by MrRiegel** [comment]
I don't think so.
I only call one block update when I change the mode of the block the redstone conduit is connected to.
And this i called in update() in TE.

**Comment by CrazyPants** [comment]
The RS conduits are not working correctly atm which is causing all sorts of issues. I am going to have a look into them when I get a chance. Hopefully whatever is happening here will be fixed in the process.

**Comment by CrazyPants** [comment]
I have just finished a cleanup of the redstone conduit code and have (hopefully) significantly improved their behaviour. Would you be able to update to the latest build and let me know if the issue is fixed? Anything version 3.0.0.45+ will have the new code. Available on curse or from jenkins

**Comment by CrazyPants** [comment]
Given lack of feedback I will assume this issue has been fixed by the rewrite of the RS conduits.
If the error still happens please re-raise with a new log etc

**Comment by xannor** [comment]
MrReigel did not provide feedback as I think I am the only one reporting and I have been away for a week. I can say that the issue persists even with updating to the most recent version. I have been posting crash reports to https://github.com/MrRiegel/Storage-Network/issues/55.

**Comment by jhc142002** [comment]
I first saw this in the pre-46 versions. Problem persisted in 46, and now I've updated to 1.10.2 and am still seeing the problem with 51. I see 55 is up now. I'll give it whirl later on.

Seems to appear when a chunk with conduits is reloaded. In my case, when the server has been player-less for some time and someone logs in. The crash report has been largely the same across versions.

[Latest Crash Report](http://pastebin.com/sQLpsgHn)
